src/events/event.h

```diff
@@ -46,10 +46,12 @@ struct AxisEvent {
 
 struct KeyEvent {
 	KeyState state;
+	std::optional<std::bitset<kModifierCount>> modifiers;
```

Comment:
I think instead of this you could make the binding part of the addBindingMethod of the EventMan